### PR TITLE
added: Firebase, login methods, signup method...

### PR DESCRIPTION
**Pull request by SoftwareDev108**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -17,8 +17,8 @@ class InvestMe extends StatelessWidget {
       routes: {
         LoginPage.id: (context) => LoginPage(),
         SignUpPage.id: (context) => SignUpPage(),
-        ConfirmationPage.id: (context) => ConfirmationPage(),
-        IntroTutorialPage.id: (context) => IntroTutorialPage(),
+        //ConfirmationPage.id: (context) => ConfirmationPage(),
```
Comment (kristynamason) [inline lib/main.dart]: Please do not comment this line out. See Discord or the appropriate comments to handle this.

File: lib/main.dart [hunk 2]
```diff
@@ -17,8 +17,8 @@ class InvestMe extends StatelessWidget {
...
+        //IntroTutorialPage.id: (context) => IntroTutorialPage(),
```
Comment (kristynamason) [inline lib/main.dart]: Please do not comment this line out. See Discord or the appropriate comments to handle this.

File: android/build.gradle [hunk 3]
```diff
@@ -8,6 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.gms:google-services:4.2.0'
```
Comment (kristynamason) [inline android/build.gradle]: Are we using Kotlin or Groovy here?

**Comment by kristynamason** [comment]
Please add a description to this pull request.

**Comment by kristynamason** [comment]
Please feel free to make the tickets that correspond with the appropriate Scrum tasks that are completed in this pull request.

File: android/build.gradle (hunk 3 above)
Comment (SoftwareDev108) [reply android/build.gradle]: Aside from the classpath 'com.google.gms:google-services:4.2.0', the classpath dependencies are automatically generated by Android Studio.

**Comment by SoftwareDev108** [comment]
Changes:
- added login methods
- added email field on signup page
- added register button on signup page
- integrated Google Firebase, specifically for the firebase_auth package
- added Google Firebase JSON file to project

**Comment by kristynamason** [comment]
Incomplete, but will pull and revise in another pull request.

**Comment by kristynamason** [comment]
Incomplete, but will pull and revise in another pull request.

**Pull request merged**